roles/confluent.variables/defaults/main.yml

```diff
@@ -106,8 +106,14 @@ archive_config_base_path: "{{ archive_destination_path }}"
 ### Boolean to have cp-ansible download the Confluent CLI
 confluent_cli_download_enabled: "{{rbac_enabled or secrets_protection_enabled}}"
 
-### Full path on hosts to install the Confluent CLI
-confluent_cli_path: /usr/local/bin/confluent
+### The path the Confluent CLI archive is expanded into.
+confluent_cli_base_path: /opt/confluent-cli
```

Comment:
I think I prefer /usr/local/bin because when I ssh to hosts the cli is in my PATH on most machines. 

Was the reason to put it in /opt to be more like the archive install? I would say leave this var as is